runelite-client/src/main/java/net/runelite/client/plugins/microbot/playerassist/loot/LootScript.java

```diff
@@ -38,12 +36,53 @@ public boolean run(PlayerAssistConfig config) {
             lootCoins(config);
             lootUntradeableItems(config);
             lootArrows(config);
-
+            lootCustomItems(config);
 
         }, 0, 200, TimeUnit.MILLISECONDS);
         return true;
     }
 
+    // parse csv string accounting for quotes and spaces and standardize to proper case
+    private List<String> parseCSVString(String csv) {
+        List<String> items = new ArrayList<>();
+        if (csv == null || csv.trim().isEmpty()) {
+            return items;
+        }
+
+        // split by comma but keep quoted strings intact
+        Pattern pattern = Pattern.compile("\"([^\"]*)\"|[^,]+");
+        Matcher matcher = pattern.matcher(csv);
+
+        while (matcher.find()) {
+            String item = matcher.group();
+            // remove quotes if present and trim whitespace
+            if (item.startsWith("\"") && item.endsWith("\"")) {
+                item = item.substring(1, item.length() - 1);
+            }
+            // trim whitespace and standardize case
+            item = item.trim();
+            if (!item.isEmpty()) {
+                // convert to proper case (first letter capital, rest lowercase)
+                item = item.substring(0, 1).toUpperCase() + item.substring(1).toLowerCase();
+                items.add(item);
+            }
+        }
+        return items;
+    }
+
+    private void lootCustomItems(PlayerAssistConfig config) {
+        if (config.customItemsToLoot() == null || config.customItemsToLoot().trim().isEmpty()) {
+            return;
+        }
+
+        List<String> itemsToLoot = parseCSVString(config.customItemsToLoot());
+        for (String itemName : itemsToLoot) {
+            if (Rs2GroundItem.loot(itemName, 1, config.attackRadius())) {
```

Comment:
You should use the `lootItemsBasedOnNames` and pass the config params from the script to the LootingParameters object, see "lootArrows" for example